src/Entities/Store.java

```diff
@@ -0,0 +1,43 @@
+package Entities;
+
+import Connections.Database;
+
+public class Store {
+    private final double totalCash;
+    private final double administrationFunds;
+    private final double salariesFunds;
+    private final double miscellaneousFunds;
+    private final double developmentFunds;
+    private final Database database;
+
+
+    public Store(Database database) {
+        this.database = database;
+        double totalCash = this.database.getTotalAmount();
+        this.totalCash = totalCash;
+        this.administrationFunds = totalCash * 0.27;
+        this.salariesFunds = totalCash * 0.54;
+        this.miscellaneousFunds = totalCash * 0.07;
```

Comment:
Let's convert these to constants. It will make the code cleaner. 

`private static final double ADMIN_RATIO = 0.27;
`
`
private static final double SALARIES_RATIO = 0.54;
`
`
private static final double MISCELLANEOUS_RATIO = 0.07;
`
`
private static final double DEVELOPMENT_FUNDS_RATIO = 1 - ADMIN_RATIO - SALARIES_RATIO - MISCELLANEOUS_RATIO ;
`